.travis.yml

```
@@ -1,6 +1,6 @@
 cache: npm
 language: node_js
 node_js:
-  - 10
+  - 12
```

Might it make sense to always use the latest stable?

```suggestion
  - node
```

---

TBH that's your call entirely. I don't mind either way.

---

I'm not sure if `node` is an alias for the latest version, or the latest LTS. Probably the latest version judging from the docs.

My intention with this PR was to move to the stable 12.x version because it's going to be LTS soon.

---

`lts/*` will always provide the most recent LTS branch, so that's also an option.